external/source/exploits/CVE-2022-2334/template.def

```diff
@@ -0,0 +1,1398 @@
+EXPORTS
+    ??0?$CLockableFlexArray@VCStaticCritSec@@@@QEAA@$$QEAV0@@Z=C:/Windows/System32/wbemcomn.??0?$CLockableFlexArray@VCStaticCritSec@@@@QEAA@$$QEAV0@@Z @1
```

Comment:
I'm not particularly familiar with this type of export table entry.  Is there a reason we're using a full path to the file that contains this function?  What happens if we run this on a host with the Windows system installed on the D: drive?  Can we somehow make this self-referential?